src/main/java/commission/rest/CommissionRestController.java

```diff
@@ -10,47 +15,81 @@
 @RestController
 @RequestMapping("/api")
 public class CommissionRestController {
-
+    private final PersonServiceImpl personService;
     private final SaleServiceImpl saleService;
+    private final CommissionServiceImpl commissionService;
```

Comment:
Nitpick: Don't use the implementation class as the type here, use the interface instead. That way the implementation can be swapped without having to change any code.

Reply:
I did the same mistake last time on the service layer.  Changed it to use the interface, but the `PaymentCalculatorService` doesnt implement any interfaces. I dont think i should create an interface for that, right? I dont see any reason for it, just want to double check.

Reply:
Yes, you are correct. Always use the most abstract class / interface that makes sense but don't create something which doesn't exist just so you can use an interface to define the type. If there is no abstract class / interface then just use the class name as the declaration.